### DISCUSSION TITLE
LongPressCommand interferes with children input reception capability

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/XamarinCommunityToolkit#1146] [Q&A] [answered]
The `TouchEffect.LongPressCommand` seems to interfere with children input reception capability, tried using `xct:TouchEffect.ShouldMakeChildrenInputTransparent="True"` without success:
```
<Frame xct:TouchEffect.LongPressCommand="{Binding LongPressCommand}">
      <Button Text="ok"/>
</Frame>
```

Is it an expected behavior or a bug? is there a workaround ?

**Comment by AndreiMisiukevich** [comment, marked as answer]
TouchEffect.ShouldMakeChildrenInputTransparent = "False" ?

**Reply by Cfun1** [reply]
I thought if not set then by default it will be set and with false value, like `InputTransparent`, I should have gave it a try anyway before assume :) thanks it's working

**Reply by AndreiMisiukevich** [reply]
Unfortunately, we cannot set it to False by default because of Android bug (see here https://github.com/xamarin/XamarinCommunityToolkit/issues/852)
